steps/validate.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+# Show Action Version
+echo "Using Rainforest GitHub Action v${RF_ACTION_VERSION}"
+
+# Ensure results directory is there
+mkdir -p results/rainforest
+
+echo "::add-mask::${RF_TOKEN}"
+
+# Define error helper
+error () {
+  echo "::error ::${1}"
+  echo "error=${1}" >> "$GITHUB_OUTPUT"
```

Comment:
Here `"$GITHUB_OUTPUT"` does not require `"${GITHUB_OUTPUT}"` ?

Reply:
I guess not 🤔 `echo "name=value >> $GITHUB_OUTPUT` is how [GitHub documents their "set output" functionality](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter) (so without braces or quotes), and ShellCheck only complains about the lack of quotes, not braces.